.github/workflows/capistrano.yml

```
@@ -46,18 +46,23 @@ jobs:
 # development and test jobs are identical - with different target environment and server keys
   development:
     if: inputs.environment == 'development'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)